Source/Plugins/Core/com.equella.core/src/com/tle/web/api/search/SearchResourceImpl.java

```diff
@@ -267,7 +268,7 @@ private DefaultSearch createSearch(
       String modifiedAfter,
       String modifiedBefore,
       String dynaCollectionCompound,
-      String status,
+      CsvList status,
```

Comment:
You'll note that by the time CsvList for collectionUuids has been passed to this function it's done the conversion to the standard `Collection` type. This is along the lines of the pattern that the first entry point marshalls the data into a standard model which can then readily be used anywhere.

So here, this should be:

```suggestion
      List<String> status,
```

This can be done rather cleanly up in `searchItems` with:

```java
Optional.ofNullable(status).map(CsvList::asList).orElse(null);
```